docs/api-docs/webhooks/webhook-events.mdx

```diff
@@ -554,9 +554,101 @@ Payload objects with the following scopes take the form that follows:
 
 Consult the [pages section of the Channel Webhooks Guide](/docs/integrations/webhooks/events/channels#pages).
 
+## Price Lists
+The following price lists webhook events fire in response to actions that affect a specific price list:
+
+| Name / Scope | Description |
+|:-------------|:------------|
+| `store/pricelist/created`  | Fires when a price list is created. | 
+| `store/pricelist/updated/` | Fires when a price list is updated. | 
+| `store/pricelist/activated/` | Fires when a price list is activated. | 
+| `store/pricelist/deactivated/` | Fires when a price list is deactivated. | 
+| `store/pricelist/deleted/` | Fires when a price list is deleted. | 
+| `store/pricelists/deleted/` | Fires when price lists are deleted. | 
+
+Payload objects with the following scopes take the form that follows:
+
+* `store/pricelist/created`
+* `store/pricelist/updated/` 
+* `store/pricelist/activated/`
+* `store/pricelist/deactivated/`
+* `store/pricelist/deleted/`
+
+
+```json filename="Example price list created payload object" showLineNumbers copy
+{
+  "scope": "store/priceList/created",
+  "store_id": "1001197568",
+  "data": {
+    "price_list_id": 5
+  },
+  "hash": "e0c298b8097a6a2f39d17e593a9b360f5b2fef7d",
+  "created_at": 1709327895,
+  "producer": "stores/{store_hash}"
+}
+```
+
+## Price list records
+
+The following price lists records webhook events fire in response to actions that affect a specific price list record:
+
+| Name / Scope | Description |
+|:-------------|:------------|
+| `store/pricelist/record/created`  | Fires when a price list is created. | 
+| `store/pricelist/records/created/` | Fires when price list records are updated. | 
+| `store/pricelist/record/updated/` | Fires when a price list record is updated. | 
+| `store/pricelist/records/updated/` | Fires when price list records are updated. | 
+| `store/pricelist/records/updated/` | Fires when price list records are updated. | 
+| `store/pricelist/record/deleted/` | Fires when a price list record is deleted. | 
+| `store/pricelist/records/deleted/` | Fires when price list records are deleted. | 
```

Comment:
Similar comment as above for the batch hooks- they all take the same form for price records.

Reply:
The batch hooks have the same payload. This is what I was able to confirm today.